src/Trips.js

```diff
@@ -101,9 +101,33 @@ class Trips {
     return presentDestinations;
   }
 
-  // getUserPendingTrips() {
-  //
-  // }
+  getTravelerPendingTrips(userId) {
+    const trips = this.getTravelerTripsById(userId);
+    console.log("USER ALL TRIPS>>>", trips)
+    let pendingTrips = [];
+    trips.forEach(trip => {
+      if (trip.status === "pending") {
+        pendingTrips.push(trip)
+      }
+    })
+    console.log("THE PENDING TRIPS>>>", pendingTrips)
+    return pendingTrips;
+  }
+
+  getTravelerPendingDestinations(userId, destinationData) {
```

Comment:
i'm thinking you could probably make this destinations method for the upcoming, current and past trips to!